### PR TITLE
Converted argparse to click for iqe integration

### PR DESCRIPTION
**Pull request by lcouzens**

### CODE REVIEW

**Comment by lcouzens** [comment]
Yeh thats all on the cards ;)

File: ocdeployer/__main__.py [hunk 1]
```diff
@@ -81,184 +93,127 @@ def get_variables_data(variables_file):
...
+              help="Ignore the 'requires' statement in config files and deploy anyway")
+@click.option("--scale-resources", type=float, default=1.0,
+              help="Factor to scale configured cpu/memory resource requests/limits by")
+@click.option("--custom-dir", "-u", default=os.path.join(os.getcwd(), "custom"),
+              help="Custom deploy scripts directory (default ./custom)")
+@click.option("--pick", "-p", default=None, help="Pick a single component from a service"
+                                                 " set and deploy that.  E.g. '-p myset/myvm'")
+@click.argument("dst_project")
+def deploy_to_project(
+    dst_project, no_confirm, secrets_local_dir, sets, all, secrets_src_project, env_file,
```
Comment (bsquizz) [inline ocdeployer/__main__.py]: Can we config the `@click.option` for `all` to store it with a different variable name? I like the `--all` parameter name but `all` is technically overriding a standard function here ;)

Comment (lcouzens) [reply ocdeployer/__main__.py]: Just pushed some changes, I missed some flags for some options like all :D I think I have fixed the all issue you mentioned too :+1:

**Comment by lcouzens** [comment]
Added the README.md changes too, thanks thats it anyway :+1:

**Pull request merged**